### PR TITLE
Ground descriptions

### PR DESCRIPTION
**Pull request by Greenjoe12345**
## Description of changes
Adds descriptions to the natural ground tiles that lacked them

## Why and what will this PR improve
The descriptions previously inherited from the base ground, which wasn't accurate for most of them.

## Authorship
Myself, Penny suggested the grass desc.

## Changelog
:cl:
add: Descriptions for natural ground turfs.
/:cl:

### CODE REVIEW

File: code/game/turfs/floors/natural/natural_grass.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /turf/floor/natural/grass
 	name = "grass"
+	desc = "You see a patch of grass. The grass is wild and seems to like it here."
```
Comment (out-of-phaze) [inline code/game/turfs/floors/natural/natural_grass.dm]:
```suggestion
	desc = "A patch of grass, growing steadily from the sun's nourishment."
```
please do not the wurm online

File: code/game/turfs/floors/natural/natural_lava.dm [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /turf/floor/natural/lava
 	name = "lava"
+	desc = "Incredibly hot molten rock, you can feel the heat radiating even from a distance.."
```
Comment (out-of-phaze) [inline code/game/turfs/floors/natural/natural_lava.dm]:
```suggestion
	desc = "Incredibly hot molten rock, you can feel the heat radiating even from a distance."
```
this should either be one period or an ellipsis, which is 3.

File: code/game/turfs/floors/natural/natural_ice.dm [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /turf/floor/natural/ice
 	name = "ice"
+	desc = "A sheet of smooth, slick ice. Be careful not to fall.."
```
Comment (out-of-phaze) [inline code/game/turfs/floors/natural/natural_ice.dm]:
```suggestion
	desc = "A sheet of smooth, slick ice. Be careful not to slip..."
```
ellipsis, and also i think 'slip' probably fits better than 'fall' here?

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

File: code/game/turfs/floors/natural/natural_grass.dm [hunk 4]
```diff
@@ -1,5 +1,6 @@
 /turf/floor/natural/grass
 	name = "grass"
+	desc = "A patch of grass, growing steadily from the sun's nourishment."
```
Comment (MistakeNot4892) [inline code/game/turfs/floors/natural/natural_grass.dm]: `the sun` is pretty specific. Nothing stopping these turfs being indoors either.

File: code/game/turfs/floors/natural/natural_grass.dm [hunk 5]
```diff
@@ -73,6 +75,7 @@
 	. = ..()
 	if(icon_state != "scorched")
 		SetName("scorched ground")
+		desc = "What was once lush grass has been reduced to burnt, ashy wastes."
```
Comment (MistakeNot4892) [inline code/game/turfs/floors/natural/natural_grass.dm]: `reduced to burnt ashes` would make more sense imo.

File: code/game/turfs/floors/natural/natural_lava.dm [hunk 6]
```diff
@@ -1,5 +1,6 @@
 /turf/floor/natural/lava
 	name = "lava"
+	desc = "Incredibly hot molten rock, you can feel the heat radiating even from a distance."
```
Comment (MistakeNot4892) [inline code/game/turfs/floors/natural/natural_lava.dm]: `Incredibly hot molten rock. You can feel the heat radiating even from a distance.`
or maybe
`A pool of incredibly hot molten rock; you can feel the heat radiating even from a distance.`

File: code/game/turfs/floors/natural/natural_rock.dm [hunk 7]
```diff
@@ -1,5 +1,6 @@
 /turf/floor/natural/rock
 	name = "rock floor"
+	desc = "A surface of rough, rocky ground."
```
Comment (MistakeNot4892) [inline code/game/turfs/floors/natural/natural_rock.dm]: `A patch` would flow better than `A surface` imo.

**Review by MistakeNot4892 — APPROVED** [review]

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**